test/runtime/configMatters/comm/atomic-error-msgs.chpl

```diff
@@ -8,7 +8,7 @@ areal.and(1);
 
 var astring: atomic string;
 
-var aint: atomic int;
+var aint: atomic int = 0;
```

Comment:
In this case, this is unfortunate to have to write `=0`.

Please create a follow-up issue to discuss the various scenarios where the users now have to specify the initial value explicitly.  For purposes other than testing, that is. :)

Reply:
Like the other test, this test is checking `=` on atomics. I initialized the variable in order to ensure that `=` continue to be called because that is what the test is checking. I don't see this as a big problem for user code. (Certainly authors of a record / library might write such tests.)